starboard/build/config/BUILDCONFIG.gn

```diff
@@ -86,22 +88,31 @@ if (is_mac) {
 # configuration.
 import("//starboard/build/platform_path.gni")
 
-import("//$starboard_path/platform_configuration/configuration.gni")
-import("//starboard/build/config/build_assertions.gni")
-
 # TODO(b/272790873): Set host_cpu to x86 for 32 bit builds.
 if (target_cpu == "x86" || target_cpu == "arm") {
   _host_toolchain_cpu = "x86"
 } else {
   _host_toolchain_cpu = host_cpu
 }
 host_toolchain = "//starboard/build/toolchain/$host_os:$_host_toolchain_cpu"
-if (sb_is_evergreen) {
-  target_toolchain = "//starboard/build/toolchain/linux:clang_lld"
+
+if (build_with_separate_cobalt_toolchain) {
+  cobalt_toolchain = "//starboard/build/toolchain/linux:clang_lld"
+  starboard_toolchain = "//$starboard_path/toolchain:starboard"
 } else {
-  target_toolchain = "//$starboard_path/toolchain:target"
+  target_platform_parts = string_split(target_platform, "-")
```

Comment:
This is obviously bad, but platform_configuration/configuration.gni currently references `current_toolchain`/`default_toolchain`, and I don't know how to remove that reference.